playwright/.gitignore

```
@@ -0,0 +1,14 @@
+node_modules/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+storageState.json
```

Sort lines alphabetically and remove duplicated entries:
```suggestion
/blob-report/
/playwright-report/
/playwright/.cache/
/test-results/
node_modules/
storageState.json
```